api/api.json

```diff
@@ -222,6 +227,31 @@
         }
       ]
     },
+    "ratingCreateForm": {
```

Comment:
I have the hidden field as a query parameter, do you think it should be in the form? I just realized that the name and description are optional so you're right we could just reuse this data structure actually. If we do reuse it though I think the hidden field should stay as a query parameter because its not relevant to ratings.